### DISCUSSION TITLE
Tina Admin - Last updated or created datetime for content discovery

### DISCUSSION THREAD

**Discussion by mitchmac** [tinacms/tinacms#2300] [Ideas]
It would be helpful when navigating content in a collection if additional metadata like the file's created or last updated timestamp could be viewed (and ideally sorted on). Is there Git metadata that allows that without requiring content modelling for such data? Is there any other metadata that could be helpful?

**Comment by jeffsee55** [comment]
I don't think this would require any content modeling, we have a `sys` key for this purpose which tells you things like `format`, `path`, `breadcrumbs`, etc. We could get all of the stat info there too, assuming the Github API also offers it.
